### ISSUE TITLE
UseRequire: false negative with braces

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#5125]
**While fixing please check related rules for the same problem: (e.g. UseRequireNotNull, UseCheckOrError, UseCheckNotNull).**

## Expected Behavior
Report on:
```kotlin
if (numColors < 1) {
	throw IllegalArgumentException("numColors must be 1 of greater")
}
```

## Observed Behavior
Zzzzzzzz, only reports on:
```kotlin
if (numColors < 1)
	throw IllegalArgumentException("numColors must be 1 of greater")
```

## Steps to Reproduce
```kotlin
private fun updateNumColors(numColors: Int) {
	if (numColors < 1) {
		throw IllegalArgumentException("numColors must be 1 of greater") // should be reported
	}
	println(numColors)
}
```
(remove `if`'s braces to get a violation)

## Context
Code formatting rule automatically adds braces (because it's good practice), braces should be supported with these awesome new rules.

## Your Environment

* Version of detekt used: 1.21.0
* Version of Gradle used (if applicable): 7.5
* Gradle scan link (add `--scan` option when running the gradle task): N/A
* Operating System and version: N/A
* Link to your project (if it's a public repository): https://github.com/TWiStErRob/net.twisterrob.colorfilters/pull/72/files
